src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -42,6 +43,16 @@ export class QuickBar extends LitElement {
 
   @internalProperty() private _commandMode = false;
 
+  @internalProperty() private _topEntityIdResult?: string;
+
+  @internalProperty() private _topCommandResult?: CommandItem;
+
+  @internalProperty() private _selectedIndex?: number;
```

Comment:
Do we ever set this?

Reply:
Ha, no. No we don't. Old code. Good catch

Reply:
Done (sorta... I basically just renamed it now that I _am_ using it to shift around the activated highlight)